src/Tickets/Commerce/Tables/Webhooks.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * The Webhooks table schema.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Tables;
+ */
+
+namespace TEC\Tickets\Commerce\Tables;
+
+use TEC\Common\Integrations\Custom_Table_Abstract as Table;
+
+/**
+ * Webhooks table schema.
+ *
+ * The table is used to store the webhooks events.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Tables;
+ */
+class Webhooks extends Table {
+	/**
+	 * The schema version.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const SCHEMA_VERSION = '0.0.1-dev';
+
+	/**
+	 * The base table name, without the table prefix.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $base_table_name = 'tec_tickets_commerce_webhooks';
+
+	/**
+	 * The table group.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $group = 'tec_tickets_commerce';
+
+	/**
+	 * The slug used to identify the custom table.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $schema_slug = 'tec-tickets-commerce-webhooks';
+
+	/**
+	 * The field that uniquely identifies a row in the table.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $uid_column = 'event_id';
+
+	/**
+	 * An array of all the columns in the table.
+	 *
+	 * @since TBD
+	 *
+	 * @var string[]
+	 */
+	public static function get_columns(): array {
+		return [
+			static::$uid_column,
+			'order_id',
+			'event_type',
+			'event_data',
+			'created_at',
+			'processed_at',
+		];
+	}
+
+	/**
+	 * Returns the table creation SQL in the format supported
+	 * by the `dbDelta` function.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The table creation SQL, in the format supported
+	 *                by the `dbDelta` function.
+	 */
+	protected function get_definition() {
+		global $wpdb;
+		$table_name      = self::table_name( true );
+		$charset_collate = $wpdb->get_charset_collate();
+		$uid_column      = self::uid_column();
+
+		return "
+			CREATE TABLE `{$table_name}` (
+				`{$uid_column}` varchar(128) NOT NULL,
+				`order_id` bigint(20) NULL,
```

Comment:
If this is a post ID, would it make sense to add a FOREIGN KEY (possibly with ON DELETE CASCADE) here?

Reply:
Done!